modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Changing applied managed cluster CRs using policies
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `musthave` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
+
+To remove the content from the managed cluster, set the `complianceType` field to `mustonlyhave`.
+In this mode, the policy ensures that the CR on the cluster is an exact match of what is specified in the policy.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have deployed a managed cluster from a hub cluster running {rh-rhacm}.
+
+* You have installed {cgu-operator-full} on the hub cluster.
+
+.Procedure
+
+. Remove the content that you no longer need from the affected CRs. In this example, the `disableDrain: false` line should be removed from the `SriovOperatorConfig` CR.
++
+.Example CR
+
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovOperatorConfig
+metadata:
+  name: default
+  namespace: openshift-sriov-network-operator
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  configDaemonNodeSelector:
+    "node-role.kubernetes.io/$mcp": ""
+  disableDrain: true <1>
+  enableInjector: true
+  enableOperatorWebhook: true
+----
+<1> Remove the `disableDrain: true` line.
+
+. Change the `complianceType` of the affected policies to `mustonlyhave` in the `group-du-sno-ranGen.yaml` file.
++
+.Example YAML
+[source,yaml]
+----
+- fileName: SriovOperatorConfig.yaml
```

Comment:
Missing YAML formatting to suggest that the field is incomplete.

Reply:
I'm not entirely sure what you mean. The these are valid YAML lines and the preview renders good too:
![image](https://github.com/openshift/openshift-docs/assets/87855859/f962a794-af02-416c-8bd4-3afecf7f4d6b)

I added `[...]` at the beginning and end to signal that it's in the middle of a long YAML file.

Reply:
Actually the [code to signal a YAML snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) is `# ...`. But, yeah, I think that is exactly what is being requested.

Reply:
Thanks, Michael. Yes, the #... syntax is correct.

Reply:
Thanks for the clarification, updated!